pkgs/development/python-modules/quantiphy/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flitBuildHook
+, pytestCheckHook
+, inform
+, parametrize-from-file
+, setuptools
+, voluptuous
+, quantiphy-eval
+, rkm-codes
+}:
+
+buildPythonPackage rec {
+  pname = "quantiphy";
+  version = "2.18";
```

```suggestion
  version = "2.18";

   disabled = pythonOlder "3.6";
```

See https://github.com/KenKundert/quantiphy/blob/master/pyproject.toml#L30. I would suggest to stick to support Python releases for all new module. Thus, `disabled = pythonOlder "3.7";`.